lib/organization.js

```diff
@@ -167,9 +166,9 @@ export class Organization {
 
     // TODO: we now just set the default voting machine, and assume it is used
     // throughout, but this assumption is not warranted
-    const settings = await getSettings();
-    if (settings.votingMachine) {
-      org.votingMachine = AbsoluteVote.at(settings.votingMachine);
+    const contracts = await getDeployedContracts();
+    if (contracts.defaultVotingMachine) {
+      org.votingMachine = AbsoluteVote.at(contracts.defaultVotingMachine);
```

Comment:
I note that `defaultVotingMachine` is not misspelled here.

Reply:
fixe